### PR TITLE
abominations can't pick anything up anymore

### PR DESCRIPTION
**Pull request by ShadowDeath6**
???

using stuff like paperbins will drop the item on the ground

:cl: ShadowDeath6
rscdel: Shambling Abominations cannot pick up items anymore.
/:cl:

### CODE REVIEW

**Comment by Super3222** [comment]
rip dreams of screeching abomination with a double energy sword

File: code/game/objects/items.dm [hunk 1]
```diff
@@ -265,6 +265,8 @@ var/global/image/fire_overlay = image("icon" = 'icons/effects/fire.dmi', "icon_s
 		return
 	if(anchored)
 		return
+	if(isabomination(user))
+		return
```
Comment (Cruix) [inline code/game/objects/items.dm]: Make this more generic

File: code/modules/mob/inventory.dm [hunk 2]
```diff
@@ -63,6 +63,8 @@
 /mob/proc/put_in_hand_check(obj/item/W)
 	if(lying && !(W.flags&ABSTRACT))
 		return 0
+	if(isabomination(src))
+		return 0
```
Comment (Cruix) [inline code/modules/mob/inventory.dm]: This too

**Comment by ghost** [comment]
honestly just make abominations a different mob instead of a race

simplemob would work, carbons work, race really doesn't

have the abomination store the ling body inside it so that when it reverts the ling is released from storage and put back in it but yeah really abomination is really ass at the moment and making it a mob is a good step

**Comment by Cruix** [comment]
You could do it the same way the wizard polymorph spell does it.

File: code/game/objects/items.dm (hunk 1 above)
Comment (ShadowDeath6) [reply code/game/objects/items.dm]: Generic?

**Comment by ShadowDeath6** [comment]
No, hippie's abomination is fucking terrible and goon's is good

**Comment by Super3222** [comment]
try using bitflags attached to the user's species.

**Pull request merged**